testing/run_tests.py

```diff
@@ -336,10 +336,13 @@ def RunJavaTests(filter, android_variant='android_debug_unopt'):
   gradle_cache_dir = os.path.join(out_dir, android_variant, 'robolectric_tests', '.gradle')
 
   test_class = filter if filter else 'io.flutter.FlutterTestSuite'
+  ignore_deprecations = 'true' if ignore_android_deprecations else ''
```

Comment:
could it be set in build.gradle directly, or is the flag helpful for some situations?

Reply:
I only added the flag in case someone wants to hide the warnings, but I can remove the flag if you don't see a need for that?

Reply:
@GaryQian any thoughts on this?